addons/sys_signal/fnc_getSignal.sqf

```diff
@@ -31,11 +31,13 @@ if (_count == 0) then {
         private _txAntenna = _x;
         {
             private _rxAntenna = _x;
+            private _model = GVAR(signalModel); // TODO: Change models on the fly if compatible (underwater, better frequency matching)
```

Comment:
Changing models on the fly happens already, no? Except if you disable that in the setting below.

Reply:
This is a note for later on. There may  be models that depend on how the map is processed but right now it is not the case.